catalog/geoscience-aus-cat/projects_geoscience-aus-cat_assets_ga_ls8c_nbart_gm_change_versions.libsonnet

```diff
@@ -0,0 +1,4 @@
+[
```

Comment:
To apply this file to dataset descriptions (to create links between them and to display a warning on the old dataset's page), you should import it into both old and new .jsonnet files. Example: https://github.com/google/earthengine-catalog/blob/main/catalog/UMD/UMD_hansen_global_forest_change_2023_v1_11.jsonnet

Reply:
Thanks Simon, I updated both the old and new .jsonnet files.

Reply:
Could you also add "version_config.version_links" to the links section in both files? See https://github.com/google/earthengine-catalog/blob/main/catalog/UMD/UMD_hansen_global_forest_change_2023_v1_11.jsonnet#L45

This is the command that will create predecessor/successor links. The checks will also ask you to add "deprecated: true" to the old dataset, as it will now have a successor.

Reply:
`version_config.version_links` was added to both files.  `'gee:status': 'deprecated'` and ` [deprecated]` were added to the old dataset file.